src/zjs_web_sockets.c

```diff
@@ -175,8 +175,6 @@ static void tcp_send(struct net_context *context, void *data, u32_t len)
         net_pkt_unref(send_pkt);
         return;
     }
-
-    net_pkt_unref(send_pkt);
```

Comment:
This is getting called multiple times, and it shouldn't be.  Removing this line fixes the issue.